Controller/Company/EditPost.php

```diff
@@ -0,0 +1,100 @@
+<?php
```

Comment:
This controller does not work. Does not save the attribute to the customer

Reply:
forgot to update routes.xml